04/styles/index.css

```diff
@@ -0,0 +1,166 @@
+@font-face {
+    font-family: 'Source Sans Pro';
+    font-style: normal;
+    font-weight: normal;
+    src: local('Source Sans Pro Regular'), local('SourceSansPro-Regular'), url('../fonts/SourceSansPro-Regular.otf') format('opentype');
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Source Sans Pro', sans-serif;
+    font-weight: normal;
+    font-style: normal;
+    background-color: #F4F4F4;
+}
+
+.header-nav {
+    background-color: #FFF;
+    text-transform: uppercase;
+}
+
+.header-nav__list,
+.header-nav__sublist {
+    list-style: none;
+}
+
+.header-nav__hamburger {
+    display: block;
+    cursor: pointer;
+    padding: 15px;
+}
+
+.header-nav__checkbox {
+    display: none;
+}
+
+.header-nav__hamburger-line {
+    display: block;
+    width: 25px;
+    height: 3px;
+    background-color: #000;
+    margin: 5px 0;
+}
+
+.header-nav {
+    display: none;
+}
+
+.header-nav__checkbox:checked~.header-nav {
+    display: block;
+}
+
+.header-nav__list>.header-nav__item>.header-nav__link,
+.header-nav__list>.header-nav__item>.header-nav__title,
```

Comment:
Lepiej nadać klasę i ją wykorzystać. Wtedy selektor CSS jest mniej skomplikowany, jest mniej zależności, co czyni go czytelniejszym :)

PS. Nie pisząc osobiście tego znacznika, trudno jest mi się zorientować na pierwszy rzut oka co to za element ;)